charts/stable/valheim/questions.yaml

```diff
@@ -9,205 +9,183 @@ questions:
 # Include{replica1}
 # Include{controllerExpertExtraArgs}
   - variable: secretEnv
-    group: "App Configuration"
-    label: "Image Secrets"
+    group: App Configuration
+    label: Image Secrets
     schema:
       additional_attrs: true
       type: dict
       attrs:
         - variable: SUPERVISOR_HTTP_USER
-          label: "SUPERVISOR_HTTP_USER"
+          label: Supervisor User
           schema:
             type: string
-            default: "admin"
+            default: ''
```

Comment:
```suggestion
            default: ""
```

Reply:
Should these defaults still be removed under the scope of this PR?